update python doc (#828)
jdxcode/rtx: docs/python.md

3.11.0
```

## Requirements

> rtx use pyenv(python-build as a plugin) to install python runtimes, you need to ensure [dependencies](https://github.com/pyenv/pyenv/wiki#suggested-build-environment) installed before installing python.

Reviewer: ```suggestion
rtx uses [python-build](https://github.com/pyenv/pyenv/tree/master/plugins/python-build) (part of pyenv) to install python runtimes, you need to ensure its [dependencies](https://github.com/pyenv/pyenv/wiki#suggested-build-environment) are installed before installing python.
```
